tox.ini

```diff
@@ -18,28 +17,20 @@ deps = -rtests/requirements.txt
        django19: -rrequirements/django19.txt
        django110: -rrequirements/django110.txt
        django111: -rrequirements/django111.txt
+       git+https://github.com/sass/libsass-python
```

Comment:
Git-revision For Great Justice here, please, if possible

Reply:
Aye, we'll clean that up before release :)